examples/07_cloudflare/tsconfig.json

```diff
@@ -9,7 +9,7 @@
     "skipLibCheck": true,
     "noUncheckedIndexedAccess": true,
     "exactOptionalPropertyTypes": true,
-    "types": ["react/experimental", "./.wrangler/types/runtime.d.ts"],
+    "types": ["./worker-configuration.d.ts"],
```

Comment:
If `"react/experimental"` is no longer required, we should remove it from all examples and fixtures (in a separate PR).

Reply:
#1382